src/apof/portal/templates/portal/topbar.html

```diff
@@ -16,7 +16,7 @@
                 </a>
             </li>
             <li class="nav-item">
-                <a class="nav-link" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" id="avatar" aria-expanded="false" style="background-image: url({% static "portal/images/avatar.jpg" %});">
+                <a class="nav-link" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" id="avatar" aria-expanded="false" style="background-image: url({{ user.profile.get_avatar }});">
```

Comment:
Please server avatar. Now it will make user to call external service about user's avatar after each page reload.